app/src/main/java/com/infomaniak/mail/utils/extensions/Extensions.kt

```diff
@@ -319,6 +319,11 @@ inline fun <reified T : RealmObject> MutableRealm.update(items: List<RealmObject
 fun MutableRealm.copyListToRealm(items: List<RealmObject>, alsoCopyManagedItems: Boolean = true) {
     items.forEach { if (alsoCopyManagedItems || !it.isManaged()) copyToRealm(it, UpdatePolicy.ALL) }
 }
+
+inline fun <reified T> RealmList<T>.replaceContent(list: List<T>) {
```

Comment:
It may be a better place to put this I think

Reply:
See here: #2256